sw/airborne/modules/gps/gps_ubx_ucenter.c

```diff
@@ -424,14 +424,25 @@ static inline void gps_ubx_ucenter_config_port(void)
     case GPS_PORT_UART2:
       UbxSend_CFG_PRT(gps_ubx_ucenter.dev,
           gps_ubx_ucenter.port_id, 0x0, 0x0,
+#ifdef BOARD_BEBOP
```

Comment:
Board specific defines should be avoided here. There should be a default mask with the possibility to redefine it if needed.